niworkflows/interfaces/freesurfer.py

```diff
@@ -205,19 +205,7 @@ def _post_run_hook(self, runtime):
             if not isdefined(lta_file):
                 continue
 
-            lines = Path(lta_file).read_text().splitlines(keepends=True)
-
-            fixed = False
-            newfile = []
-            for line in lines:
-                if line.startswith('filename = ') and len(line.strip("\n")) >= 255:
-                    fixed = True
-                    newfile.append('filename = path_too_long\n')
-                else:
-                    newfile.append(line)
-
-            if fixed:
-                Path(lta_file).write_text(''.join(newfile))
+        fix_lta_length(lta_file)
```

Comment:
This should be inside the for loop.

```suggestion
            fix_lta_length(lta_file)
```